recipes-kernel/linux/linux-altera-lts_5.4.bb

```diff
@@ -3,7 +3,7 @@ LINUX_VERSION_SUFFIX = "-lts"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
 
-SRCREV = "7299712b69b1869ddaf1c143c9264b69351d9e2b"
+SRCREV = "be87fdd9e66c1bdea063b7771af74fdb2b389571"
```

Comment:
should it also bump LINUX_VERSION on line 1?

Reply:
I am checking the test result of new version bump, will submit another pull request for version bump

Reply:
thanks I will wait for you you to report back